designer/server/src/main/scala/pl/touk/nussknacker/ui/api/TestingApiErrorMessages.scala

```diff
@@ -0,0 +1,41 @@
+package pl.touk.nussknacker.ui.api
+
+object TestingApiErrorMessages {
+
+  val noSourcesWithTestDataGeneration = "No sources with test data generation available"
+
+  val noDataGenerated = "No data was generated"
+
+  val noInputRecords = "No input records found"
```

Comment:
Can you group messages by two categories:
1) Messages related by providing sample data by us 
2) Messages related with data passed by user
?
It took me a while to understand what is the difference between noInputRecords vs noDataGenerated, tooManyCharactersGenerated vs tooManyCharactersReceived